addons/web/static/src/core/utils/colors.js

```diff
@@ -198,6 +198,80 @@ export function convertCSSColorToRgba(cssColor) {
     // Note: however, if ever implemented be careful of 'white'/'black' which
     // actually are color names for our color system...
 
+    // Check if cssColor is a color() functional notation allowing colorspace with implicit sRGB.
+
+    // First, define the formal syntax with regular expressions
+
+    const number = `([+-]?([0-9]+([.][0-9]*)?|[.][0-9]+))`;
+    const percentage = `(${number}%)`;
+
+    // <alpha-value> =
+    //   <number> | <percentage>
+
+    const alphaValue = `(${percentage}|${number})`;
+
+    // <predefined-rgb> =
+    //   srgb          |
+    //   srgb-linear   |
+    //   display-p3    |
+    //   a98-rgb       |
+    //   prophoto-rgb  |
+    //   rec2020
+
+    const predefinedRgb = `(?<predefined_rgb>srgb|srgb\\-linear|display\\-p3|a98\\-rgb|prophoto\\-rgb|rec2020)`;
+
+    // <predefined-rgb-params> =
+    //   <predefined-rgb> [ <number> | <percentage> | none ]{3}
+
+    const predefinedRgbParams = `(${predefinedRgb})\\s+(?<c1>${alphaValue}|none)\\s+(?<c2>${alphaValue}|none)\\s+(?<c3>${alphaValue}|none)`;
+
+    // <xyz-space> =
+    //   xyz      |
+    //   xyz-d50  |
+    //   xyz-d65
+
+    const xyzSpace = `(?<xyz_space>xyz|xyz\\-d50|xyz\\-d65)`;
+
+    // <xyz-params> =
+    //   <xyz-space> [ <number> | <percentage> | none ]{3}
+
+    const xyzParams = `(${xyzSpace})\\s+(?<cc1>${alphaValue}|none)\\s+(?<cc2>${alphaValue}|none)\\s+(?<cc3>${alphaValue}|none)`;
+
+    // <colorspace-params> =
+    //   <predefined-rgb-params>  |
+    //   <xyz-params></xyz-params>
+
+    const colorspaceParams = `(?<colorspace_params>${predefinedRgbParams}|${xyzParams})`;
+
+    // <slash> = /
+
+    const slash = `/`;
+
+    // <color()> =
+    //  color( <colorspace-params> [ / [ <alpha-value> | none ] ]? )
+
+    const color = `color\\(${colorspaceParams}(\\s+${slash}\\s+(?<A>${alphaValue}|none))?\\);?`;
+    const regex = new RegExp(color);
+
+    if (regex.exec(cssColor) !== null) {
+        // We can use regex.exec(cssColor).groups to actually perform the real calculations to
+        // convert color spaces to RGBa. But this trick is relatively robust. We'll see if we
+        // can define self-contained conversions if it doesn't fit to performance goals.
+
+        const cvs = document.createElement("canvas");
```

Comment:
Every variable that store an HTMLElement should have a name finishing with `El` so here const `canvasEl`.

---
Extra infos:
- If the variable is an array of HTMLElement its name should finish with `Els`.
- If the variable stores a JQuery element, its name should start with `$`.